tests/api/v1/samples/get.js

```diff
@@ -83,6 +87,79 @@ describe(`tests/api/v1/samples/get.js, GET ${path} >`, () => {
     .end(done);
   });
 
+  it('get with wildcard and with cacheGetSamplesWildcard flag on ' +
+    'shold cache response', (done) => {
+    tu.toggleOverride('cacheGetSamplesByNameWildcard', true);
+    api.get(`${path}?name=${sampleName}*`)
+    .set('Authorization', token)
+    .expect(constants.httpStatus.OK)
+    .end((err, res) => {
+      if (err) {
+        return done(err);
+      }
+
+      redisCache.get(`${sampleName}*`, (cacheErr, reply) => {
+        if (cacheErr || !reply) {
+          return done(cacheErr);
+        }
+
+        expect(JSON.parse(reply).length).to.be.above(ZERO);
+        expect(JSON.parse(reply)[0].name).to.equal(`${sampleName}`);
+        redisCache.del(`${sampleName}*`);
+
+        tu.toggleOverride('cacheGetSamplesByNameWildcard',
```

Comment:
Putting it here means the value doesn't get restored to its original value if we returned done from line 98 or line 103. Saving the previous value of a feature toggle and restoring that original value should go in before/after blocks.

Reply:
But I don't need in each and every test. So there is no point of it writing in before / after.

Reply:
The only way to make sure they get called no matter the outcome of the test is to use before/after. Put the ones which need it in their own nested "describe" and add the before/after in that describe only.